src/components/RefinementList/__tests__/RefinementListItem-test.tsx

```diff
@@ -2,10 +2,27 @@
 
 import { h } from 'preact';
 import { shallow } from 'enzyme';
+import { ReactElementLike } from 'prop-types';
 import RefinementListItem from '../RefinementListItem';
 
+type Props = {
+  facetValue: string;
+  facetValueToRefine: string;
+  isRefined: boolean;
+  handleClick: jest.Mock<any, any>;
+  className: string;
+  templateData: {
+    template: string;
+  };
+  templateKey: string;
+  templateProps: {
+    template: string;
+  };
+  subItems: h.JSX.Element;
+};
+
 describe('RefinementListItem', () => {
-  const props = {
+  const props: Props = {
```

Comment:
or I think the type here likely could be omitted. What doesn't work if it's removed?

Reply:
Everything works, but in `render(askedProps)` `askedProps` is set as `any` and I tried to avoid implicit `any` in the files I converted.

So if I need to type `askedProps` at least also type `props` to keep consistency.